website/content/docs/agent/autoauth/methods/azure.mdx

```diff
@@ -17,3 +17,11 @@ on the value of the `resource` parameter.
 - `role` `(string: required)` - The role to authenticate against on Vault
 
 - `resource` `(string: required)` - The resource name to use when getting instance information
+
+- `object_id` `(string: optional)` - The object ID of the user-assigned managed identity to use
+  when acquiring an [access token](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-to-use-vm-token#get-a-token-using-http).
+  This parameter is interchangeable with `client_id`. Only one of `object_id` or `client_id` may be provided.
+
+- `client_id` `(string: optional)` - The client ID of the user-assigned managed identity to use
+  when acquiring an [access token](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-to-use-vm-token#get-a-token-using-http).
+  This parameter is interchangeable with `object_id`. Only one of `object_id` or `client_id` may be provided.
```

Comment:
Are these parameters actually interchangeable? They do seem to be mutually exclusive, but when looking at this on Azure, my managed identity has both a Client ID and Object (principal) ID, and they're different uuid's.

From testing this in an Azure VM, it does appear that `principal_id` and `object_id` are interchangeable, but not `object_id` and `client_id`.

```suggestion
- `object_id` `(string: optional)` - The object ID of the user-assigned managed identity to use
  when acquiring an [access token](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-to-use-vm-token#get-a-token-using-http).
  Only one of `object_id` or `client_id` may be provided.

- `client_id` `(string: optional)` - The client ID of the user-assigned managed identity to use
  when acquiring an [access token](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-to-use-vm-token#get-a-token-using-http).
  Only one of `object_id` or `client_id` may be provided.
```